RELEASE_NOTES.md

```diff
@@ -19,14 +19,18 @@ You may need to change your `.ruby-version` file too.
 
 If not, you need to adapt it to your environment, for instance by changing the decidim docker image to use ruby:3.x.x.
 
-### 1.2. Update your Gemfile
+### 1.2. Update your application configuration
+
+In this version we are changing Decidim's underlaying configuration engine, so, in order to update your application, make sure you read changes about the environment variables (section 3.5 )
```

Comment:
I'm proposing a change of the title. Also I like to have copy paste here to ease up the update process

```suggestion
In this version we are changing Decidim's underlaying configuration engine, so, in order to update your application, make sure you read changes about the environment variables (read more about it at "3.4 Deprecation of `Rails.application.secrets`").

```bash
git rm config/secrets.yml
git rm config/initializers/decidim.rb
wget https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb -O config/storage.yml
```
```